src/middleware.ts

```diff
@@ -44,13 +46,27 @@ export const expressAggressiveCache = (options?: Options) => {
 
   const responseBucket = store<CachedResponse>();
   const chunkBucket = store<Chunk>();
+  const cacheKeyBucket = store<string>();
 
-  const purge: PurgeFunction = async (tag: string) => {
+  const purge: PurgeFunction = async (cacheTag: string) => {
     throw new Error(
-      `purge for tag ${tag} not implemented - API could still change - do not use`
+      `purge for cache tag ${cacheTag} not implemented - API could still change - do not use`
     );
   };
 
+  const updateCacheKeyBucketOptional = async (
+    req: Request,
+    res: ExtendedResponse,
+    cacheKey: string
+  ) => {
+    if (getCacheTag != undefined) {
```

Comment:
!= undefined is redundant here

Reply:
pas exactement, je voulais vraiment la réponse d'API undefined pas "" par exemple
j'aurais du utiliser !== comme ailleur dans la librairie